### PR TITLE
Add method to return CDN domain values

### PR DESCRIPTION
**Pull request by shomix**
This PR will:
- Add a new method `getCdnDnsRecords ` to the `storage`, `site` and `compute` sdk's. This method will return values which need to be added to the domain record so that it can be verified. 
  - `cdnARecords ` - will contain the value that should be added for domains
  - `cdnCnameRecords ` - will contain the value that should be added for subdomain.
- Update the README of storage SDK. Removed all the methods and added a link to the docs. https://github.com/spheronFdn/sdk/tree/SPH-2176/packages/storage

### CODE REVIEW

File: packages/storage/src/index.ts [hunk 1]
```diff
@@ -219,6 +216,18 @@ export class SpheronClient {
     );
   }
 
+  async getCdnValues(): Promise<{
```
Comment (rekpero) [inline packages/storage/src/index.ts:219]:
```suggestion
  async getCdnDnsRecords(): Promise<{
```

File: packages/storage/src/index.ts [hunk 2]
```diff
@@ -219,6 +216,18 @@ export class SpheronClient {
...
+    cdnDomainValue: string;
+    cdnSubdomainValue: string;
+  }> {
+    const { recordIpv4V2, recordCnameV2 } =
+      await this.spheronApi.getCdnRecords();
+    return {
+      cdnDomainValue: recordIpv4V2,
```
Comment (rekpero) [inline packages/storage/src/index.ts:226]:
```suggestion
      cdnARecords: recordIpv4V2,
```

File: packages/storage/src/index.ts [hunk 3]
```diff
@@ -219,6 +216,18 @@ export class SpheronClient {
...
+      cdnSubdomainValue: recordCnameV2,
```
Comment (rekpero) [inline packages/storage/src/index.ts:227]:
```suggestion
      cdnCnameRecords: recordCnameV2,
```

**Review by rekpero — CHANGES_REQUESTED** [review]
check my comments

File: packages/storage/src/index.ts (hunk 1 above)
Comment (shomix) [reply packages/storage/src/index.ts]: Updated

File: packages/storage/src/index.ts (hunk 2 above)
Comment (shomix) [reply packages/storage/src/index.ts]: Updated.

**Comment by shomix** [comment]
@rekpero added the `getCdnDnsRecords` to site and compute as well.

**Review by rekpero — DISMISSED** [review]

File: packages/site/package.json [hunk 4]
```diff
@@ -35,7 +35,7 @@
     "dist/"
   ],
   "dependencies": {
-    "@spheron/core": "1.0.5",
+    "@spheron/core": "1.0.7",
```
Comment (dusanstanis) [inline packages/site/package.json:38]: 1.0.8 here ? Yesterday we released 1.0.7

File: packages/compute/package.json [hunk 5]
```diff
@@ -33,7 +33,7 @@
     "dist/"
   ],
   "dependencies": {
-    "@spheron/core": "^1.0.4"
+    "@spheron/core": "^1.0.7"
```
[the remark this answers is not in the document]
Comment (shomix) [reply packages/compute/package.json]: yep, fixed.

File: packages/site/package.json (hunk 4 above)
Comment (shomix) [reply packages/site/package.json]: yep, fixed.

**Review by rekpero — APPROVED** [review]

**Review by rekpero — APPROVED** [review]

**Review by dusanstanis — APPROVED** [review]

**Pull request merged**